src/pybind/mgr/dashboard/frontend/src/app/shared/directives/form-name.directive.ts

```diff
@@ -0,0 +1,8 @@
+import { Directive, Input } from '@angular/core';
+
+@Directive({
+  selector: '[cdFormName]'
+})
+export class FormNameDirective {
```

Comment:
Since we use the term `scope` in some parts of the dashboard, we should also use it here.
`Name`  gives the wrong idea. 
```suggestion
export class FormScopeDirective {
```

Reply:
Done.